pkg/ebpftracer/decoder/decoder.go

```diff
@@ -20,23 +21,39 @@ import (
 	"github.com/google/gopacket/layers"
 )
 
+var sensitiveValuePattern = regexp.MustCompile(`(?i)(password|passwd|pass|pwd|secret|token|key|creds|credential)(=?)`)
```

Comment:
In such case why not just set this default regex on flags.

Reply:
because right now we have two different branches for user-provided regex and default regex. The default regex takes into account the keyword and redacts the value, e.g. `--password=abc123` -> `--password=<redacted>`, while the user-provided one will redact the whole string it matches, e.g. `--password=abc123` -> `<redacted>`.